### PR TITLE
Add jquery.markitup w/ git auto-update

### PR DESCRIPTION
**Pull request by spiro79**
Add jquery.markitup w/ git auto-update

### CODE REVIEW

File: packages/j/jquery.markitup.json [hunk 1]
```diff
@@ -0,0 +1,38 @@
...
+    "url": "https://github.com/markitup/1.x.git"
+  },
+  "autoupdate": {
+    "source": "git",
+    "target": "https://github.com/markitup/1.x.git",
+    "fileMap": [
+      {
+        "basePath": "markitup",
+        "files": [
+          "**/*.+(js|css|png|html)"
```
Comment (MattIPv4) [inline packages/j/jquery.markitup.json:27]:
```suggestion
          "**/*.@(js|css|png)"
```
The CDN doesn't host HTML files + use `@` to match one item in the group

**Review by MattIPv4 — CHANGES_REQUESTED** [review]
Ty :)

File: packages/j/jquery.markitup.json (hunk 1 above)
Comment (spiro79) [reply packages/j/jquery.markitup.json]: Done.

**Review by MattIPv4 — APPROVED** [review]
Lgtm, tyvm!

**Pull request merged**